packages/react-kitchen-sink/src/schema/firestoreDataConverter.ts

```
@@ -1,4 +1,4 @@
-import { type DocumentData, GeoPoint, type QueryDocumentSnapshot, Timestamp } from 'firebase/firestore'
+import { type DocumentData, GeoPoint, type QueryDocumentSnapshot, Timestamp } from '@firebase/firestore'
```

ESLint

[error] Filename is not in kebab case. Rename it to `firestore-data-converter.ts`.

(unicorn/filename-case)